modules/shared/src/main/scala/org/tessellation/statechannel/validation.scala

```diff
@@ -0,0 +1,14 @@
+package org.tessellation.statechannel
+
+import derevo.cats.eqv
+import derevo.derive
+
+object validation {
+
+  @derive(eqv)
+  sealed trait StateChannelValidation
```

Comment:
Longer but maybe better: `StateChannelValidationType` or `StateChannelValidationMode`

Reply:
changing to `StateChannelValidationType` :+1: